scripts/createSampleYAML.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
This would be a good candidate for a test in Travis CI. All you need is a directory in the proper structure, so you could  make some empty files